src/components/App/App.tsx

```diff
@@ -0,0 +1,10 @@
+import { Image } from "../Image/Image";
+function App() {
+  return (
+    <div className="app">
```

Comment:
Ideally we keep `App` clean and only add what we need here since it's such a central component. For this PR, we can remove `App` and leave testing to the storybook.

Reply:
That makes sense thank you!

Reply:
I reverted main.tsx back to the version in main and removed App.tsx.